Automattic/jetpack
# WAF: Ensure mode is set during activation

## Proposed changes:

* Moves the functionality that ensures certain options exist from the `on_waf_activation` method (which runs on module activation) into the generic `Waf_Runner::activate` method (which runs on module activation, plus firewall settings updates).
* Fixes the case where a site has no "mode" option set, preventing the firewall from falling back into "silent" mode.

### Other information:


## Jetpack product discussion



N/A

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Test the basic WAF functionality.
* With the firewall active, delete the `jetpack_waf_mode` option from your site.
* Change one of the firewall settings.
* Verify the `jetpack_waf_mode` option has been reset to "normal".
